apps/core-app/src/components/DaoTable.tsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import { useTable, Column, UseTableRowProps } from 'react-table';
+import styled from 'styled-components';
+import { indigoDark } from '@radix-ui/colors';
+
+import { ITransformedMembership } from '@daohaus/dao-data';
+import { ProfileAvatar, Tag } from '@daohaus/ui';
+import {
+  charLimit,
+  getNetworkName,
+  readableNumbers,
+  truncateAddress,
+} from '@daohaus/common-utilities';
+
+// REVIEW NOTES
+// Can this be refactored to use DaoHaus Table?
```

Comment:
It probably can be.

You have the prep the columns and table data up above and the pass a few other things:
- array of sortable column ids
- load more method
- sort method

We could track this in a new refactor ticket